exammode/models.py

```diff
@@ -0,0 +1,120 @@
+from django.db import models
+from django.utils import timezone
+
+from course.models import CourseInstance, CourseModule
+from exercise.exercise_models import LearningObject
+from userprofile.models import UserProfile
+
+# Create your models here.
+
+
+class ExamSessionManager(models.Manager):
+
+    def get_queryset(self):
+        initial_queryset = super().get_queryset()
+        queryset = [q for q in initial_queryset if (
+            q.can_start <= timezone.now())]
+
+        return queryset
+
+    def is_active(self):
+        initial_queryset = super().get_queryset()
+        queryset = [q for q in initial_queryset if (q.can_start <= timezone.now(
+        ) and (timezone.now() <= q.can_start + timezone.timedelta(hours=q.duration)))]
+
+        return queryset
+
+
+class ExamSession(models.Model):
+    """
+    Represents one instance of a course exam
+    """
+
+    course_instance = models.ForeignKey(
+        CourseInstance, on_delete=models.CASCADE)
+    exam_module = models.ForeignKey(
+        CourseModule, on_delete=models.CASCADE, null=True)
```

Comment:
Does `exam_module` need `null=True` when it can not be empty (`blank` is False by default)?

Reply:
Should have also blank=True to function as intended